osi_hostvehicledata.proto

```diff
@@ -319,7 +320,7 @@ message HostVehicleDataClass
     //
     // \brief This message contains all the necessary information of the localization solution.
     //
-    message VehicleLocalization
+    message VehicleGEOPosition
```

Comment:
Could it be, that xyz will be also part of this message, so georeferenced is not the only option in the future? Look to depricated optional BaseMoving location_rmse = 2;
So localization would be more open..

Reply:
@ThomasNaderBMW : as discussed, I would keep xyz and geo-position messages separately for flexibility purposes.

Reply:
I would like xyz will be keep too, as there are many simulator models that working with local xyz coordination systems. 
There are currently three geo-reference formats available in OpenDRIVE, I think the z-axis basically refers to ellipsoidal height, and the other coordinate system the z-axis refers to sea lavel elevation.

As an additional question, I can't know the NDS Map specifications, but are the geo-reference specifications used fixed in NDS?
Does the newly definition of VehicleGEOPosition in the OSI message have a goal that the referenced geo-reference specification is fixed? (e.g use NDS's geo-reference?)

Reply:
Sry for the late reply.
I gave it a new structure, can you both have a look?
To the questions:
- NDS information shall be possible to be transported
- proj4-String of OpenDrive has to be discussed as it is used to calculate the WGS84-Position (End-Result) as an Input-Parameter. Is this information necessary beside the simulation-tool, that is parsing the OpenDrive?